### PR TITLE
mungebot: Blunderbuss selects multiple assignees

### PR DESCRIPTION
**Pull request by apelisse**
Select 2 assignees by default now (rather than just 1) in case one of
the two is out-of-office or just too busy.

Note that this is not going to add an extra-assignee to pull-requests
that already have one (only works if the pull-request doesn't
have any assignee yet).

### CODE REVIEW

**Comment by apelisse** [comment]
Potentially fixes https://github.com/kubernetes/contrib/issues/1688 @fejta @bgrant0607

File: mungegithub/mungers/blunderbuss.go [hunk 1]
```diff
@@ -159,16 +193,12 @@ func (b *BlunderbussMunger) Munge(obj *github.MungeObject) {
...
-	selection := rand.Int63n(weightSum)
-	owner := ""
-	for o, w := range potentialOwners {
-		owner = o
-		selection -= w
-		if selection <= 0 {
-			break
-		}
+
+	owners := selectMultipleOwners(potentialOwners, weightSum, 2)
```
Comment (grodrigues3) [inline mungegithub/mungers/blunderbuss.go]: should the two be hardcoded or should it be 

> b.NumberAssignees

**Comment by fejta** [comment]
Yay!!!

**Comment by lavalamp** [comment]
Won't this result in everyone waiting for the other person to do the review? Sorry I didn't notice this until now. I guess we can wait and see.

**Comment by fejta** [comment]
Agree we cannot predict how well this will/won't work. We'll continue iterating on other ideas if this won't doesn't work.

**Pull request merged**